omod/src/main/java/org/openmrs/module/patientflags/extension/html/FlagPatientDashboardHeaderExt.java

```diff
@@ -43,14 +43,9 @@ public String getOverrideContent(String test) {
 		
 		if (!results.isEmpty()) {
 			String content = "<TABLE><TR><TD>";
-			
-			// comment out the header text for now
-			//        + Context.getMessageSourceService().getMessage("patientflags.flagpatientdashboard.header") + " ";
-			
 			for (Flag flag : results) {
-				content = content + "<SPAN " + (flag.getPriority() != null ? flag.getPriority().getStyle() : "") + ">" + flag.getLocalizedMessage() + "</SPAN>, ";
+				content = content + "<SPAN " + (flag.getPriority() != null ? flag.getPriority().getStyle() : "") + ">" + flag.getName() + "</SPAN>, ";

```

Comment:
Basically I think you should be able to just add a getLocalizedName() method to the Flag object that follows the same pattern as getLocalizedMessage()